ror/.stylelintrc.json

```diff
@@ -3,7 +3,11 @@
   "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
   "rules": {
     "at-rule-no-unknown": null,
-    "scss/at-rule-no-unknown": true,
+    "scss/at-rule-no-unknown": [true,
+                                {
+                                  "ignoreAtRules": ["tailwind", "apply", "variants", "responsive", "screen"]
+                                }
+                               ],
```

Comment:
I think indentation needs some improvements

Reply:
Thanks, @ahangarha , 
I did it according to MV's previous `./stylelintrc.json` indentation style.
Let me know if you have any suggestions.

Reply:
@od-c0d3r - Thank you.
we could use [Prettier - Code formatter](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) VSCode extension.